### ISSUE TITLE
Major breaking change as of 3.1.0 – cannot run on Google App Engine (No module named _ctypes)

### ISSUE THREAD

**Issue by Drarok** [Julian/jsonschema#614]
The change introduced in 6a7155a854d57e3f13f9dec15f2ade0820dbca2a, "Switch to the newer importlib_metadata" introduces a new dependency (`importlib_metadata`) which uses `ctypes` which are _not_ available on [Google App Engine][1]:

```
  File "…/jsonschema/__init__.py", line 31, in <module>
    import importlib_metadata
  File "…/importlib_metadata/__init__.py", line 15, in <module>
    from ._compat import (
  File "…/importlib_metadata/_compat.py", line 30, in <module>
    import pathlib2 as pathlib
  File "…/pathlib2/__init__.py", line 5, in <module>
    import ctypes
  File "…/.pyenv/versions/2.7.16/lib/python2.7/ctypes/__init__.py", line 7, in <module>
    from _ctypes import Union, Structure, Array
  File "…/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/python/runtime/sandbox.py", line 1120, in load_module
    raise ImportError('No module named %s' % fullname)
ImportError: No module named _ctypes
```

Since it was only a minor version bump, it was picked up as a valid dependency but totally broke our development environment which took a _lot_ of hunting to find.

I appreciate that as a project you might not want to revert this change for such a niche use-case, but hopefully other people will find this issue and it'll help them resolve the problem.

In short, __if you're using this library* on Google App Engine, pin to v3.0.2__.

\*(or a project with this as a dependency such as [Flask-RESTPlus][2])

[1]: https://cloud.google.com/appengine/kb/?csw=1
[2]: https://flask-restplus.readthedocs.io/en/stable/
